lib/rpmsg/rpmsg.c

```diff
@@ -37,8 +37,6 @@ static int rpmsg_get_buffer_size(struct rpmsg_virtio_device *rvdev)
 {
 	int length;
 
-	metal_mutex_acquire(&rvdev->lock);
```

Comment:
Curious why to move the lock?

Reply:
This is just helper function used internally only, the caller can provide locking.
But I just notice than the buffer size for the master side is wrong, for now, for both master and slave, we only support fixed the size